Assets/Tests/InputSystem/Plugins/DualShockTests.cs

```diff
@@ -158,6 +158,18 @@ public void Devices_SupportsDualShockAsHID_WithJustPIDAndVID()
         });
 
         Assert.That(device, Is.AssignableTo<DualShockGamepad>());
+
+        device = InputSystem.AddDevice(new InputDeviceDescription
+        {
+            interfaceName = "HID",
+            capabilities = new HID.HIDDeviceDescriptor
+            {
+                vendorId = 0x54C,
+                productId = 0x5C4,
+            }.ToJson()
+        });
+
+        Assert.That(device, Is.AssignableTo<DualShockGamepad>());
```

Comment:
Recommend using `[TestCase]` for this one.

```
    [Test]
    [Category("Devices")]
    [TestCase(0x54C, 0x9CC)]
    [TestCase(0x54C, 0x5C4)]
    public void Devices_SupportsDualShockAsHID_WithJustPIDAndVID(int vendorId, int productId)
    {
        var device = InputSystem.AddDevice(new InputDeviceDescription
        {
            interfaceName = "HID",
            capabilities = new HID.HIDDeviceDescriptor
            {
                vendorId = vendorId,
                productId = productId,
            }.ToJson()
        });

        Assert.That(device, Is.AssignableTo<DualShockGamepad>());
    }
```

Reply:
fixed